### DISCUSSION TITLE
Timestamp to epoch function/pattern?

### DISCUSSION THREAD

**Discussion by ajfriend** [duckdb/duckdb#4492] [Q&A] [answered]
DuckDB has `epoch_ms(ms)` and `to_timestamp(sec)` functions to convert from time since the epoch to a timestamp: https://duckdb.org/docs/sql/functions/timestamp

Is there a function to do the reverse? Or is there an existing pattern to do that in DuckDB?

**Comment by Alex-Monahan** [comment, marked as answer]
Yes!
```sql
epoch(timestamp_column)
```

**Comment by ajfriend** [comment]
Ah, ok. I hadn't previously spotted it on this page of the docs: https://duckdb.org/docs/sql/functions/datepart

Thanks!
